src/components/icon/icon.stories.tsx

```diff
@@ -87,6 +87,32 @@ export const VariousBgSizes: ComponentStory<typeof Icon> = () => {
   );
 };
 
+export const BgSizesAndFontSizes: ComponentStory<typeof Icon> = () => {
```

Comment:
**suggestion** (non-blocking): I'd consider adding a story where the `fontSize` value is larger than that of `bgSize` too